sw/applications/src/net_fusedconv.c

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <math.h>
+
+#include "conv2d.h"
+#include "data_fusedconv.h"
+#include "printf.h"
+#include "snrt.h"
+#include "utils.h"
+
+int main() {
+    float *ptr = (void *)snrt_cluster_memory().start;
```

Comment:
you assume that there is nothing else at TCDM start. There is `snrt_l1alloc` which also performs simple size-checking and ensures proper alignment.